<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>advancedViewClass</class>
 <widget class="QWidget" name="advancedViewClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>811</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>advancedView</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">/*QWidget {
	background: black;
    border-radius: 5px;
}
*/
QWidget#backgroundWidget {
	background: rgb( 128, 128, 128);
	border-radius: 5px;
}



/****************
 *         v3.0            *
 ****************/


/* --------- General font and layout defaults ----------*/

QDialog {
 font: 18px &quot;DinPRO-regular&quot;;
 color: white;
 background-color: rgb( 128, 128, 128);
 border: 1px solid rgb( 128, 128, 128);
 border-radius: 5px;
}

QWidget {
 font: 13px &quot;DinPRO-regular&quot;;
}

QLabel {
 font: 14px &quot;DinPRO-regular&quot;;
}

QLabel#titlebarLabel {
	background: rgb( 51, 51, 51 );
    border-top-left-radius: 5px;
    border-top-right-radius: 5px;
}
QLabel#titlebarTextLabel {
    font: 16pt &quot;DINPro-Medium&quot;;
    color: white;
}

QLabel#instructionsLabel
{
    background: rgb( 191, 191, 191 );
}

QLabel#instructionsTextLabel
{
    font: 12pt &quot;DINPro-Medium&quot;;
}

QLabel#wideDivideLabel
{
    background: rgb( 191, 191, 191 );
}

QLabel#wideDivideTextLabel
{
    font: 12pt &quot;DINPro-Medium&quot;;
}


/* ---------------- Pushbuttons ----------------*/

QPushButton 
{
    background: rgb( 52, 73, 94 );
    border: 1px solid rgb( 52, 73, 94 );
    color: white;
    font: 12pt &quot;DinPRO-Medium&quot;;
    text-align: center;
    border-radius: 10px;
    border-width: 2px;
}

QPushButton::hover 
{
    background: rgb(52, 73, 94 );
    border-color: rgb( 166, 166, 166 );
    color: rgb( 214, 214, 214);
    border-width: 1px;
    border-radius: 10px;
}

 QPushButton:default 
{
     background: rgb( 51, 51, 51 );
/*     border: 1px solid black; /* make the default button prominent */
/*	 font: 17px &quot;DinPRO-bold&quot;;*/
 }

QPushButton:disabled 
{
     background:  rgb( 51, 51, 51 ); 
     border: rgb( 51, 51, 51 ); 
 }

QPushButton:checked 
{
    background:  rgb(  191, 191, 191  ); 
    border: rgb(  191, 191, 191  ); 
    color: rgb( 52, 73, 94 );
 }

/* --------- Spin Boxes -----------*/

QAbstractSpinBox {
	font: 18px DinPRO-medium;
	border: 0;
    border-radius: 5px;
}

QAbstractSpinBox::up-button {
	border: 0;
   background: rgba(233,233,235,255);
   border-top-right-radius: 8px;
   margin-top: -2px;
   margin-bottom: -1px;
   padding-bottom: -5px;
   width: 15px;
}

QAbstractSpinBox::up-arrow  {
  image: url(:/images/arrow_up_normal.png);
}

QAbstractSpinBox::up-arrow:hover {
  image: url(:images/arrow_up_rollover.png);
}

QAbstractSpinBox::down-button {
   border: 0;
   background: rgba(233,233,235,255);
   border-bottom-right-radius: 8px;
   padding-top: -5px;
   margin-bottom: -2px;
   margin-top: -1px;
   width: 15px;
}

QAbstractSpinBox::down-arrow:hover {
  image: url(:/images/arrow_down_rollover.png);
}

QAbstractSpinBox::down-arrow {
  image: url(:/images/arrow_down_normal.png);
}

/* group boxes */


/* ------------- Line edits and list views ---------------*/

QLineEdit, QListView {
    color: #808080;
	font: 18px DinPro-medium;
    selection-color: white;   
    border: 2px groove white;
    border-radius: 4px;
}


/* --------- Check Boxes ----------*/

QCheckBox {
	spacing: 5px;
}

QCheckBox::indicator {
	border:   rgb( 128, 128, 128 );
    background:   rgb( 128, 128, 128 );
	width: 20px;
	height: 20px;
	margin-top: 4px;
}

QCheckBox::indicator:unchecked {
	image: url(:images/checkBox_normal.png);
}

QCheckBox::indicator:checked {
	image: url(:images/checkBox_active.png);
}

QDialogButtonBox
{
   button-layout: 1;
}


/* ---------- Group Boxes ---------- */

QGroupBox::title {
     subcontrol-origin: margin;
     subcontrol-position: top left; /* position at the top center */
     left: 10px;
     padding: 1px;
 }

 QGroupBox {
    font: 14.75pt DINPro-medium;
    color: rgba(210, 210, 210, 255);
     background-color:    rgb( 128, 128, 128 );;
     border-radius: 5px;
     margin-top: 1ex;
    padding-top: 3px;
 }</string>
  </property>
  <widget class="QWidget" name="backgroundWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>815</width>
     <height>811</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>619</y>
      <width>271</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="fullCaseRecordingLabel">
       <property name="font">
        <font>
         <family>DinPRO-regular</family>
         <pointsize>-1</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Full Case Recording:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>DinPRO-regular</family>
         <pointsize>-1</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Version:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="linesPerSecondLabel">
       <property name="font">
        <font>
         <family>DinPRO-regular</family>
         <pointsize>-1</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>N/A</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <family>DinPRO-regular</family>
         <pointsize>-1</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>DAQ Updates Per Second:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>DinPRO-regular</family>
         <pointsize>-1</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>DAQ Level: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="fullCaseRecordingValueLabel">
       <property name="font">
        <font>
         <family>DinPRO-regular</family>
         <pointsize>-1</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>DISABLED</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="sledModeValuelabel">
       <property name="text">
        <string>Unknown</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="daqUpdatesPerSecondLabel">
       <property name="font">
        <font>
         <family>DinPRO-regular</family>
         <pointsize>-1</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>N/A</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="versionLabel">
       <property name="font">
        <font>
         <family>DinPRO-regular</family>
         <pointsize>-1</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>x.y.z</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <family>DinPRO-regular</family>
         <pointsize>-1</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Updates Per Second: </string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="sledModeLabel">
       <property name="text">
        <string>Sled Mode:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="daqLevelLabel">
       <property name="font">
        <font>
         <family>DinPRO-regular</family>
         <pointsize>-1</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>N/A</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="fwVersionLabel">
       <property name="text">
        <string>FW Versions:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="fwVersionVal">
       <property name="text">
        <string>N/A</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="tdcCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>780</y>
      <width>224</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Trigger on Motor Top-Dead-Center</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="saveSignalsPushButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>34</y>
      <width>130</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Signals</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="evoaBox">
    <property name="geometry">
     <rect>
      <x>461</x>
      <y>632</y>
      <width>351</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Imaging Strength</string>
    </property>
    <widget class="QPushButton" name="evoaSetDefaultButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Return to Default</string>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>138</width>
       <height>44</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="evoaStatusLabel">
        <property name="font">
         <font>
          <family>DinPRO-regular</family>
          <pointsize>-1</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Status:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="evoaStatusVal">
        <property name="font">
         <font>
          <family>DinPRO-regular</family>
          <pointsize>-1</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Disabled</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="evoaVoltageLabel">
        <property name="font">
         <font>
          <family>DinPRO-regular</family>
          <pointsize>-1</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Voltage (V):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="evoaVoltageVal_v">
        <property name="font">
         <font>
          <family>DinPRO-regular</family>
          <pointsize>-1</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>0.0V</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="levelGauge" name="evoaControlWidget" native="true">
     <property name="geometry">
      <rect>
       <x>222</x>
       <y>3</y>
       <width>97</width>
       <height>105</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="titlebarLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>815</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="titlebarTextLabel">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>-5</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>ADVANCED VIEW</string>
    </property>
   </widget>
   <widget class="QLabel" name="instructionsLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>33</y>
      <width>815</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
    background: rgb(191, 191, 191);
    font: 12pt &quot;DINPro-Medium&quot;;
    padding-left: 15px;
    color: black;
}
</string>
    </property>
    <property name="text">
     <string>RAW OCT SIGNAL</string>
    </property>
   </widget>
   <widget class="dataPlotWidget" name="rawDataPlot" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>66</y>
      <width>815</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb( 117, 117, 117 );
color: rgb(0, 0, 0 );
</string>
    </property>
   </widget>
   <widget class="QLabel" name="wideDivideLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>267</y>
      <width>815</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
    background: rgb(191, 191, 191);
    font: 12pt &quot;DINPro-Medium&quot;;
    padding-left: 15px;
    color: black;
}
</string>
    </property>
    <property name="text">
     <string>PROCESSED SIGNAL</string>
    </property>
   </widget>
   <widget class="dataPlotWidget" name="fftDataPlot" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>815</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb( 117, 117, 117 );
color: rgb(0, 0, 0 );
</string>
    </property>
   </widget>
   <widget class="QLabel" name="highSpeedLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>580</y>
      <width>815</width>
      <height>39</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
    background: rgb(191, 191, 191);
    font: 12pt &quot;DINPro-Medium&quot;;
    padding-left: 15px;
    color: black;
}
</string>
    </property>
    <property name="text">
     <string>SYSTEM PARAMETERS</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="laserBox">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>760</y>
      <width>331</width>
      <height>43</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="formLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>331</width>
       <height>43</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="laserLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="1">
       <widget class="QProgressBar" name="laserDiodeStatus">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>30</horstretch>
          <verstretch>40</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>161</width>
          <height>38</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>151</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QProgressBar {
	border-radius: 0;
	background: rgb( 51, 51, 51 );
	border: solid rgb( 128, 128, 128 );
}

QProgressBar::chunk { 
    image: url(:/images/progress-chunk-on.png);
    margin: 4px;
}
</string>
        </property>
        <property name="value">
         <number>50</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="laserDiodeButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>60</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>161</width>
          <height>41</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Turn the diode on or off.</string>
        </property>
        <property name="text">
         <string>Laser Diode</string>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>10</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Ctrl+Shift+L</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QCheckBox" name="swEncoder">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>770</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>SW Encoder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getSledStatusButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>660</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Status</string>
    </property>
   </widget>
   <zorder>gridLayoutWidget</zorder>
   <zorder>tdcCheckBox</zorder>
   <zorder>evoaBox</zorder>
   <zorder>titlebarLabel</zorder>
   <zorder>titlebarTextLabel</zorder>
   <zorder>instructionsLabel</zorder>
   <zorder>rawDataPlot</zorder>
   <zorder>wideDivideLabel</zorder>
   <zorder>fftDataPlot</zorder>
   <zorder>saveSignalsPushButton</zorder>
   <zorder>highSpeedLabel</zorder>
   <zorder>laserBox</zorder>
   <zorder>swEncoder</zorder>
   <zorder>getSledStatusButton</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>levelGauge</class>
   <extends>QWidget</extends>
   <header>levelgauge.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>dataPlotWidget</class>
   <extends>QWidget</extends>
   <header>dataPlotWidget.h</header>
   <container>1</container>
   <slots>
    <signal>updateBrightness(int)</signal>
    <signal>updateContrast(int)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
